packages/codd/lib/codd_common.py

```
@@ -530,6 +572,42 @@ def normalize_check_level(value: Any) -> str:
     return level
 
 
+def _as_glob_list(value: Any, field_name: str) -> list[str]:
+    """scope/code_scope の include・exclude を glob 文字列のリストへ正規化する。
+
+    YAML でリスト記法を忘れて単一文字列（例: ``code_scope.include: "src/**/*.py"``）を
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)